src/System.Private.ServiceModel/tests/Common/Infrastructure/src/Infrastructure.Common.csproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+    <PropertyGroup>
+        <MinimumVisualStudioVersion>11.0</MinimumVisualStudioVersion>
+        <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+        <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+        <OutputType>Library</OutputType>
+        <AppDesignerFolder>Properties</AppDesignerFolder>
+        <RootNamespace>Infrastructure.Common</RootNamespace>
+        <AssemblyName>Infrastructure.Common</AssemblyName>
+        <SignAssembly>false</SignAssembly>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+        <ProjectGuid>{AFD69665-89A3-42AE-A32E-AB2CBBE6EE7E}</ProjectGuid>
+        <CLSCompliant>false</CLSCompliant>
+    </PropertyGroup>
+    <ItemGroup>
+        <Compile Include="**\*.cs" />
+    </ItemGroup>
+    <ItemGroup>
+        <None Include="project.json" />
+        <None Include="testproperties.props" />
+        <None Include="testproperties.targets" />
+    </ItemGroup>
+    <ItemGroup>
+        <!-- Compile tests against the contract, but copy our local-built implementation for testing -->
+        <ProjectReference Include='$(WcfSourceProj)'>
+            <Project>{9e50e7bf-cd6e-4269-a6dd-59fd0bd6c0fd}</Project>
+            <Name>System.Private.ServiceModel</Name>
+            <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+            <OutputItemType>Content</OutputItemType>
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+            <Targets>Build;DebugSymbolsProjectOutputGroup</Targets>
+        </ProjectReference>
+    </ItemGroup>
+    <Import Project="testproperties.props" />
+    <Import Project="testproperties.targets" />
+    <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+    <!-- Add our code-gen target after the imports above because they reset $(CoreCompileDependsOn) -->
+    <PropertyGroup>
+        <CoreCompileDependsOn>$(CoreCompileDependsOn);CreateTestProperties</CoreCompileDependsOn>
+    </PropertyGroup>
+</Project>
```

Comment:
Nit: add newline at the end.  Not needed for this PR but next time you touch the file